packages/calling/src/CallSettings/CallSettings.ts

```diff
@@ -83,20 +94,23 @@ export class CallSettings implements ICallSettings {
 
   /**
    * Updates DND setting at the backend.
+   * @param enabled - true to enable DND, false to disable DND.
    */
   public async setDoNotDisturbSetting(enabled: boolean): Promise<CallSettingResponse> {
     return this.backendConnector.setDoNotDisturbSetting(enabled);
   }
 
   /**
    * Reads Call Forward setting at the backend.
+   *
    */
   public async getCallForwardSetting(): Promise<CallSettingResponse> {
     return this.backendConnector.getCallForwardSetting();
   }
 
   /**
    * Updates Call Forward setting at the backend.
+   * @param callForwardingRequest - CallForwardSetting object.
```

Comment:
Are we exporting the type for `CallForwardSetting`? If not, we need to be more descriptive here